src/openzaak/conf/dev.py

```diff
@@ -9,7 +9,9 @@
 from urllib3.exceptions import SystemTimeWarning
 
 os.environ.setdefault("DEBUG", "yes")
-os.environ.setdefault("ALLOWED_HOSTS", "*")
+if "test" not in sys.argv:
+    os.environ.setdefault("ALLOWED_HOSTS", "*")
```

Comment:
Why is this needed?

Reply:
I forgot to add a comment, setting `ALLOWED_HOSTS` to `*` causes issues with django-loose-fk, I'll see if I can pin down the failing tests and add an override_settings for ALLOWED_HOSTS instead

Reply:
removed this and added `override_settings` where needed